versions.tf

```diff
@@ -1,4 +1,33 @@
 
 terraform {
-  required_version = ">= 0.12"
+  required_version = ">= 0.13"
+  required_providers {
+    acme = {
+      source = "terraform-providers/acme"
+    }
+    archive = {
+      source = "hashicorp/archive"
+    }
+    aws = {
+      source = "hashicorp/aws"
```

Comment:
I think the change is good, but I wonder if we can version pin to the major or minor version some providers, in particular the AWS provider. WDYT @ianstanton?

Reply:
I don't see an issue with that. This was generated by `terraform`, so I'll add those and see if it causes any unexpected issues.